python/cudf/cudf/tests/test_udf_masked_ops.py

```diff
@@ -682,6 +690,391 @@ def f(x):
     assert precompiled.currsize == 1
 
 
+@string_udf_test
+@pytest.mark.parametrize(
+    "data", [{"str_col": ["cudf", "rapids", "AI", "gpu", "2022"]}]
+)
+def test_string_udf_len(data):
+    # tests the `len` function in string udfs
+    data = cudf.DataFrame(data)
+
+    def func(row):
+        st = row["str_col"]
+        return len(st)
+
+    run_masked_udf_test(func, data, check_dtype=False)
+
+
+@string_udf_test
+@pytest.mark.parametrize(
+    "data", [{"str_col": ["cudf", "rapids", "AI", "gpu", "2022", "cuDF"]}]
+)
+@pytest.mark.parametrize("substr", ["a", "cu", "2"])
+def test_string_udf_startswith(data, substr):
+    # tests the `startswith` method of strings
+    data = cudf.DataFrame(data)
+
+    def func(row):
+        st = row["str_col"]
+        return st.startswith(substr)
+
+    run_masked_udf_test(func, data, check_dtype=False)
+
+
+@string_udf_test
+@pytest.mark.parametrize(
+    "data",
+    [
+        {
+            "str_col": [
+                "cudf",
+                "rapids",
+                "AI",
+                "gpu",
+                "2022",
+                "cuDF",
+                "again_gpu",
+            ]
+        }
+    ],
+)
+@pytest.mark.parametrize("substr", ["a", "gpu", "2"])
+def test_string_udf_endswith(data, substr):
+    # tests the `endswith` method of strings
+    data = cudf.DataFrame(data)
+
+    def func(row):
+        st = row["str_col"]
+        return st.endswith(substr)
+
+    run_masked_udf_test(func, data, check_dtype=False)
+
+
+@string_udf_test
+@pytest.mark.parametrize(
+    "data",
+    [
+        {
+            "str_col": [
+                "cudf",
+                "rapids",
+                "AI",
+                "gpu",
+                "2022",
+                "cuDF",
+                "again_gpu",
+            ]
+        }
+    ],
+)
+@pytest.mark.parametrize("substr", ["u", "gpu", "a"])
+def test_string_udf_find(data, substr):
+    # tests the `find` method of strings
+    data = cudf.DataFrame(data)
+
+    def func(row):
+        st = row["str_col"]
+        return st.find(substr)
+
+    run_masked_udf_test(func, data, check_dtype=False)
+
+
+@string_udf_test
+@pytest.mark.parametrize(
+    "data",
+    [
+        {
+            "str_col": [
+                "cudf",
+                "rapids",
+                "AI",
+                "gpu",
+                "2022",
+                "cuDF",
+                "again_gpu",
+            ]
+        }
+    ],
+)
+@pytest.mark.parametrize("substr", ["u", "gpu", "a"])
+def test_string_udf_rfind(data, substr):
+    # tests the `find` method of strings
```

Comment:
These comments are not necessary (the test name is enough). Later tests in this file do not have comments like this one, so I'd just remove them uniformly. Every additional line makes for a potential bug/typo as well. 😅
```suggestion
    # tests the `rfind` method of strings
```